data/assets/scene/solarsystem/planets/earth/layers/watermasks/modis_water_mask.asset

```diff
@@ -6,8 +7,15 @@ local layer = {
   FilePath = asset.localResource("modis_water_mask.wms")
 }
 
-asset.onInitialize(function () 
-  openspace.globebrowsing.addLayer(globeIdentifier, "WaterMasks", layer)
-end)
+assetHelper.registerLayerAndExport(asset, layer, globeIdentifier, "WaterMasks")
 
-asset.export("layer", layer)
+
+asset.meta = {
+  Name = "MODIS Water Mask",
+  Version = "1.0",
+  Description = [[ MODIS Water  Mask layer for Earth globe.]],
```

Comment:
nitpicky, but the other assets wrote `WaterMask` together as one word

Reply:
I think writing them "Water Mask" would be better than together